React/CalcApp/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "calc-app",
+  "version": "0.0.0",
+  "license": "MIT",
+  "scripts": {
+    "start": "nx serve",
+    "build": "nx build",
+    "test": "nx test"
+  },
+  "private": true,
+  "dependencies": {
+    "@swc/helpers": "~0.5.0",
+    "react": "18.2.0",
+    "react-dom": "18.2.0"
+  },
+  "devDependencies": {
```

Comment:
can you please add jest and react testing library to the project?

Reply:
react testing library was already there. I added jest